packages/web-js-sdk/src/enhancers/withPersistTokens/index.ts

```diff
@@ -40,31 +41,45 @@ export const withPersistTokens =
 
     const afterRequest: AfterRequestHook = async (_req, res) => {
       if (res?.status === 401) {
-        clearTokens();
+        clearTokens(storagePrefix);
       } else {
         persistTokens(
           await getAuthInfoFromResponse(res),
-          sessionTokenViaCookie
+          sessionTokenViaCookie,
+          storagePrefix
         );
       }
     };
 
-    const sdk = createSdk(addHooks(config, { beforeRequest, afterRequest }));
+    const sdk = createSdk(
+      addHooks(config, {
+        beforeRequest: beforeRequest(storagePrefix),
+        afterRequest,
+      })
+    );
 
-    const wrappedSdk = wrapWith(sdk, ['logout', 'logoutAll'], wrapper);
+    const wrappedSdk = wrapWith(
+      sdk,
+      ['logout', 'logoutAll'],
+      wrapper(storagePrefix)
+    );
+
+    const refreshToken = () => getRefreshToken(storagePrefix);
+    const sessionToken = () => getSessionToken(storagePrefix);
 
     return Object.assign(wrappedSdk, {
-      getRefreshToken,
-      getSessionToken,
+      getRefreshToken: refreshToken,
+      getSessionToken: sessionToken,
     }) as any;
   };
 
-const wrapper: SdkFnWrapper<{}> =
+const wrapper =
+  (prefix?: string): SdkFnWrapper<{}> =>
   (fn) =>
   async (...args) => {
     const resp = await fn(...args);
 
-    clearTokens();
+    clearTokens(prefix);
```

Comment:
not critical, but why we need to create another scope here for the prefix and pass it here?
![image](https://github.com/descope/descope-js/assets/10514677/edb07c15-0ec3-4a6d-815b-29338549a752)

we can just use `storagePrefix` here

Reply:
how? the `wrapper` is an external function. i can declare it inside the component and then i dont need the extra scope

Reply:
disregard, I thought I deleted this comment before review 😅